src/lib.rs

```diff
@@ -296,6 +385,14 @@ impl PanCam {
 impl Default for PanCam {
     fn default() -> Self {
         Self {
+            move_mode: MoveMode::Keyboard,
```

Comment:
my other thought is: should this mode be the default? zooming is still done by scrolling, so this feels off

Reply:
As I agree with johanhelsing that we should just implement simultaneous movement as the only mode I'll just remove `MoveMode` tomorrow when I update the pr.